packages/manager/modules/vps/package.json

```diff
@@ -21,6 +21,7 @@
   "peerDependencies": {
     "@ovh-ux/manager-advices": "^0.0.0 || ^1.0.0",
     "@ovh-ux/manager-billing": "^2.0.0",
+    "@ovh-ux/manager-catalog-price": "^1.0.0",
```

Comment:
Not needed to add it in both apps `dedicated` and `vps` ?

Reply:
Has to be listed in project dependency to be used inside module.

Reply:
Yes but here it is declared as a `peer`, we might need to add it as a dependency to apps using this module (like https://github.com/ovh/manager/blob/master/packages/manager/apps/vps/package.json for instance)